docs/gettingstarted/debugprofiler.md

```diff
@@ -0,0 +1,62 @@
+# Debug Profiler
+
+Minecraft provides a Debug Profiler that can be used to find time consuming code. Specially considering things like `TickEvents` and Ticking `TileEntities` this can be very usefull for modders and server owners that want to find a lag source.
+
+## Using the Debug Profiler
+
+The Debug Profiler is very simple to use, it requires two commands `/debug start` and `/debug stop`
+They are booth pretty self explaining, but you run `/debug start` to start the profiling process and then run `/debug stop` to stop it again.
+The important part here is that the more time you give it to collect the data the better the results will be.
+It is recommended to at least let it collect data for a minute.
+
+!!! note
+  In case you want to test your `TileEntities` you need to place them into the world.
```

Comment:
You could say the same about blocks and entities. It is quite self explanatory. I think this note should be generalized or removed.